modules/post/windows/manage/ducky.rb

```diff
@@ -0,0 +1,140 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows Ducky Script Parser',
+        'Description' => %q{
+          This module can be used to execute supplied *lightweight* Ducky Scripts - only supports: GUI, STRING, and STRINGLN.
+          It also supports sending individual keyevents based on Metasploit's current standards.
+
+          Setting VERBOSE to true will output each line of your file as it is parsed.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'dru1d <tyler.booth[at]cdw.com>',
+        ],
+        'References' => [
+          'URL', 'https://docs.hak5.org/hak5-usb-rubber-ducky/duckyscript-tm-quick-reference',
+        ],
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
```

Comment:
```suggestion
          'Stability' => [],
```

Let's remove this, as the duckyscript could potentially crash the target

Reply:
I will remove the CRASH_SAFE tag per your recommendation.